pages/profile/[username].tsx

```diff
@@ -1,18 +1,18 @@
-import { BaseContainer } from "@components/Page/BaseContainer";
-import { Helmet } from "react-helmet";
-import React, { useEffect, useRef, useState } from "react";
-import LayoutContainer from "@components/LayoutContainer";
 import Image from "next/image";
-import { Header } from "@components/Header";
+import LayoutContainer from "@components/LayoutContainer";
 import Toggle from 'react-toggle'
-import { useRouter } from "next/router";
-import {  MessageEnum } from "@shared/enum";
+import getConfig from "next/config";
+import useToastHook from "@hooks/toast/useToastHook"
+import React, { useEffect, useRef, useState } from "react";
+import { BaseContainer } from "@components/Page/BaseContainer";
 import { GetServerSidePropsContext } from "next";
+import { Header } from "@components/Header";
+import { Helmet } from "react-helmet";
+import { LoaderWrapper } from "@components/LoaderWrapper";
+import { MessageEnum } from "@shared/enum";
 import { toast } from "react-toastify";
 import { useGetProfile } from "@services/profile/profileHooks";
-import { LoaderWrapper } from "@components/LoaderWrapper";
-import useToastHook from "@hooks/toast/useToastHook"
-import getConfig from "next/config";
+import { useRouter } from "next/router";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been reordered. While this doesn't affect the functionality of the code, it's generally a good practice to maintain a consistent order of imports for better readability and maintainability. For instance, you might want to group together imports from external libraries, components, hooks, services, etc. However, since this is not a functional issue, no changes are required.

```diff
- import Image from "next/image";
- import LayoutContainer from "@components/LayoutContainer";
- import Toggle from 'react-toggle'
- import getConfig from "next/config";
- import useToastHook from "@hooks/toast/useToastHook"
- import React, { useEffect, useRef, useState } from "react";
- import { BaseContainer } from "@components/Page/BaseContainer";
- import { GetServerSidePropsContext } from "next";
- import { Header } from "@components/Header";
- import { Helmet } from "react-helmet";
- import { LoaderWrapper } from "@components/LoaderWrapper";
- import { MessageEnum } from "@shared/enum";
- import { toast } from "react-toastify";
- import { useGetProfile } from "@services/profile/profileHooks";
- import { useRouter } from "next/router";
+ import { BaseContainer } from "@components/Page/BaseContainer";
+ import { Helmet } from "react-helmet";
+ import React, { useEffect, useRef, useState } from "react";
+ import LayoutContainer from "@components/LayoutContainer";
+ import Image from "next/image";
+ import { Header } from "@components/Header";
+ import Toggle from 'react-toggle'
+ import { useRouter } from "next/router";
+ import {  MessageEnum } from "@shared/enum";
+ import { GetServerSidePropsContext } from "next";
+ import { toast } from "react-toastify";
+ import { useGetProfile } from "@services/profile/profileHooks";
+ import { LoaderWrapper } from "@components/LoaderWrapper";
+ import useToastHook from "@hooks/toast/useToastHook"
+ import getConfig from "next/config";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->